spec.html

```diff
@@ -4078,36 +4078,36 @@ <h1>Type Conversion</h1>
+      <h1>ToPrimitive ( _input_ [ , _preferredType_ ] )</h1>
+      <p>The abstract operation ToPrimitive takes argument _input_ and optional argument _preferredType_. It converts its _input_ argument to a non-Object type. If an object is capable of converting to more than one primitive type, it may use the optional hint _preferredType_ to favour that type. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _input_ is an ECMAScript language value.
         1. If Type(_input_) is Object, then
-          1. If _PreferredType_ is not present, let _hint_ be *"default"*.
-          1. Else if _PreferredType_ is hint String, let _hint_ be *"string"*.
-          1. Else,
-            1. Assert: _PreferredType_ is hint Number.
-            1. Let _hint_ be *"number"*.
           1. Let _exoticToPrim_ be ? GetMethod(_input_, @@toPrimitive).
           1. If _exoticToPrim_ is not *undefined*, then
+            1. If _preferredType_ is not present, let _hint_ be *"default"*.
+            1. Else if _preferredType_ is ~string~, let _hint_ be *"string"*.
+            1. Else,
+              1. Assert: _preferredType_ is ~number~.
+              1. Let _hint_ be *"number"*.
             1. Let _result_ be ? Call(_exoticToPrim_, _input_, &laquo; _hint_ &raquo;).
             1. If Type(_result_) is not Object, return _result_.
             1. Throw a *TypeError* exception.
-          1. If _hint_ is *"default"*, set _hint_ to *"number"*.
-          1. Return ? OrdinaryToPrimitive(_input_, _hint_).
+          1. If _preferredType_ is not present, let _preferredType_ be ~number~.
+          1. Return ? OrdinaryToPrimitive(_input_, _preferredType_).
         1. Return _input_.
       </emu-alg>
       <emu-note>
-        <p>When ToPrimitive is called with no hint, then it generally behaves as if the hint were Number. However, objects may over-ride this behaviour by defining a @@toPrimitive method. Of the objects defined in this specification only Date objects (see <emu-xref href="#sec-date.prototype-@@toprimitive"></emu-xref>) and Symbol objects (see <emu-xref href="#sec-symbol.prototype-@@toprimitive"></emu-xref>) over-ride the default ToPrimitive behaviour. Date objects treat no hint as if the hint were String.</p>
+        <p>When ToPrimitive is called with no hint, then it generally behaves as if the hint was ~number~. However, objects may over-ride this behaviour by defining a @@toPrimitive method. Of the objects defined in this specification only Date objects (see <emu-xref href="#sec-date.prototype-@@toprimitive"></emu-xref>) and Symbol objects (see <emu-xref href="#sec-symbol.prototype-@@toprimitive"></emu-xref>) over-ride the default ToPrimitive behaviour. Date objects treat no hint as if the hint was ~string~.</p>
```

Comment:
I notice you also changed "as if the hint _were_"  to "as if the hint _was_". Both seem correct to me, so I don't care either way, just want to call attention to it.

Reply:
(Here, "were" uses the [subjunctive mood](https://en.wikipedia.org/wiki/English_subjunctive), which seems to be gradually fading from use in English.)

Reply:
I lightly prefer "were".

Reply:
Reverted back to "were".